examples/editor/examples/basic/App.tsx

```diff
@@ -153,14 +108,19 @@ export function App() {
     <BlockNoteView className="root" editor={editor}>
       <FormattingToolbarPositioner editor={editor} />
       <HyperlinkToolbarPositioner editor={editor} />
-      <SlashMenuPositioner editor={editor} />
       <SideMenuPositioner editor={editor} />
+      <DefaultPositionedSuggestionMenu editor={editor} />
```

Comment:
What do you think about the following?

```
<BlockNoteView ... >
    <SuggestionMenu
        editor={editor}
        triggerCharacter={"@"}
        getItems={(query, closeMenu, clearQuery) =>
          getMentionMenuItems(editor, query, closeMenu, clearQuery)
        }
      />
   <DefaultBlockNoteUIElements formattingToolbar={false} slashMenu={true} ...etc... /> // props default to true
</BlockNoteView>
```

This way, `DefaultBlockNoteUIElements` can be used to toggle on / off built-in UI elements

Reply:
Yeah I think this is way better, shortens basically every use case except for removing most elements (which I think is pretty uncommon).